index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>My Home</title>
+    </head>
+    <body>
+        <p>Hello</p>
+    </body>
+</html>
```

Comment:
```suggestion
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <title>My Home</title>
    </head>
    <body>
        <p>Hello</p>
    </body>
</html>
```
이 부분의 코드에서 어떤 걸 더 추가하실 건가요?